reference/5.1/PowershellGet/Install-Script.md

```diff
@@ -89,21 +89,21 @@ PS C:\> Install-Script -Repository "Local1" -Name "Required-Script3" -Scope "All
 PS C:\> Get-InstalledScript -Name "Required-Script3"
 Version    Name                                Type       Repository           Description
 -------    ----                                ----       ----------           -----------
-2.5        Required-Script3                    Script     local1               Description for the Required-Script3 script PS C:\> Get-InstalledScript -Name "Required-Script3" | Format-List * 
+2.5        Required-Script3                    Script     local1               Description for the Required-Script3 script PS C:\> Get-InstalledScript -Name "Required-Script3" | Format-List *
 Name                       : Required-Script3
 Version                    : 2.5
 Type                       : Script
 Description                : Description for the Required-Script3 script
 Author                     : pattif
-CompanyName                : 
-Copyright                  : Â© 2015 Microsoft Corporation. All rights reserved. 
+CompanyName                :
+Copyright                  : ???? 2015 Microsoft Corporation. All rights reserved.
```

Comment:
mangled conversion here

Reply:
I thought I got them all. Somehow I missed that one.